aspnetcore/signalr/introduction-signalr-core.md

```diff
@@ -0,0 +1,60 @@
+---
+uid: signalr/introduction-to-signalr
```

Comment:
These were made consistent across the repo on a recent PR to the order: title first, uid last, and everything else in the middle alphabetized.

Here's a cut-'n-paste if you like (and I added the `ms.custom` entry) ...

```
title: Introduction to SignalR on ASP.NET Core
author: rachelappel
description: An overview of SignalR on ASP.NET Core
manager: wpickett
ms.author: rachelap
ms.custom: mvc
ms.date: 2/19/2018
ms.prod: aspnet-core
ms.technology: dotnet-signalr
ms.topic: article
uid: signalr/introduction-to-signalr
```